event/src/main/java/com/bazaarvoice/emodb/event/db/astyanax/DefaultSlabAllocator.java

```diff
@@ -71,8 +74,31 @@ private static ScheduledExecutorService defaultExecutor(LifeCycleRegistry lifeCy
         return executor;
     }
 
+    // Compute how many slots in a slab will be used to do a default (new slab) allocation, and how many bytes it will consume
+    static Pair<Integer, Integer> defaultAllocationCount(Pair<Integer,Integer> amountOfSlabUsed, PeekingIterator<Integer> eventSizes) {
```

Comment:
Separate from the return value of `Pair`, one suggestion is to replace the `amountOfSlabUsed` parameter with two separate integer parameters, which may make them easier to identify, rather than `left` and `right`. I'm flexible on this, just though I would give the suggestion.